components/ModEditSubmissionTopbar.vue

```diff
@@ -76,4 +75,8 @@ const acceptSubmission = () => {
 const rejectSubmission = () => {
     console.log('You are rejected')
 }
+
+watch(selectedSubmissionId, newSelectedId => {
```

Comment:
uhhh....it's watching and setting itself 😆

Reply:
Whoops